app/agents/voice/breeze_buddy/services/telephony/Exotel/exotel.py

```
@@ -37,7 +37,16 @@ def make_call(self, customer_mobile_number: str, outbound_number: str):
         try:
-            resp = requests.post(url, data=payload)
+            # Build proxy configuration if available
+            proxies = None
+            if self.config.PROXY_HOST and self.config.PROXY_PORT:
+                proxy_url = f"http://{self.config.PROXY_HOST}:{self.config.PROXY_PORT}"
+                proxies = {
+                    'https': proxy_url
+                }
+                logger.info(f"Using proxy: {proxy_url}")
+            
+            resp = requests.post(url, data=payload, proxies=proxies)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)